src/main/java/com/resources/errorhandling/HttpExceptionMessage.java

```diff
@@ -1,5 +1,6 @@
 package com.resources.errorhandling;
 
 public class HttpExceptionMessage {
-	public static final String IDNOTFOUND = "ID not found.";
-}
\ No newline at end of file
+		public static final String IDNOTFOUND = "The record with the ID: " + new ObjectId() + " was not found.";
```

Comment:
Yor implementation is not working it supposed that this message return the current resourceId not a new one.